js/bitmax.js

```diff
@@ -52,73 +57,92 @@ module.exports = class bitmax extends Exchange {
             'urls': {
                 'logo': 'https://user-images.githubusercontent.com/1294454/66820319-19710880-ef49-11e9-8fbe-16be62a11992.jpg',
                 'api': 'https://bitmax.io',
-                'test': 'https://bitmax-test.io',
+                'test': 'https://bitmax-test.io/api',
                 'www': 'https://bitmax.io',
                 'doc': [
-                    'https://github.com/bitmax-exchange/api-doc/blob/master/bitmax-api-doc-v1.2.md',
+                    'https://bitmax-exchange.github.io/bitmax-pro-api/#rest-apis',
                 ],
                 'fees': 'https://bitmax.io/#/feeRate/tradeRate',
-                'referral': 'https://bitmax.io/#/register?inviteCode=EL6BXBQM',
+                'referral': 'https://bitmax.io/#/register?inviteCode=T6J9R0EB',
             },
             'api': {
                 'public': {
                     'get': [
                         'assets',
+                        'barhist',
+                        'barhist/info',
+                        'cash/assets',
+                        'cash/products',
                         'depth',
                         'fees',
```

Comment:
Looks like there's no `fees` endpoint – https://bitmax.io/api/pro/v1/fees

Reply:
fee rate is a little complicated now
https://bitmax.io/#/feerate/transactionfee-traderate